### ISSUE TITLE
[FEATURE] : Improve navbar user experience

### ISSUE THREAD

**Issue by sathwikgaddi** [PritamSarbajna/tourism-website#106]
In the responsiveness design, when a list item is clicked in navbar menu button, the navbar is not closing automatically closing and we have to manually close it. It feels a little uncomfortable for user experience.

**Comment by sathwikgaddi** [comment]
Also, if you find that it needs to be enhanced, could you please assign this to me?!

**Comment by Anshika029** [comment]
Can you please elaborate on this more?

**Comment by Shreyashkshirsagar** [comment]
We've noticed your Enhancement issue titled "Improving Navbar User Experience" on your GitHub repository. Our expertise would greatly benefit this project, and we kindly request your assistance in addressing this issue.
Could you please consider collaborating with us on this enhancement? our insights and contributions would be highly valuable.
I Want you to give access to this and direct us to work on it.
Looking forward to your response.

**Comment by PritamSarbajna** [comment]
Yeah sure @Shreyashkshirsagar. If you want to work on it, I can assign you.

**Comment by BurhanAhmad0** [comment]
> In the responsiveness design, when a list item is clicked in navbar menu button, the navbar is not closing automatically, we have to manually close it. It feels a little uncomfortable for user experience.

Yeah, that's irritating
Assign me this job I'll fix this

**Comment by jesse-maurice** [comment]
I have already fixed it and submitted a pull request which is yet to be  checked and closed.
@PritamSarbajna @sathwikgaddi

**Comment by BurhanAhmad0** [comment]
Oh, Okay
